dev-portal/src/utils/test-utils.jsx

```diff
@@ -29,3 +29,24 @@ export const renderWithRouter = (
  */
 export const resolveAfter = (ms, resolution) =>
   new Promise(resolve => setTimeout(() => resolve(resolution), ms))
+
+/**
+ * Suppress React 16.8 act() warnings globally.
+ * The React team's fix won't be out of alpha until 16.9.0.
```

Comment:
Note that 16.9.0 is out, and breaks nothing while obviating the need for this warning suppression utility. However, our Semantic UI React dependency (and perhaps others) uses features deprecated in 16.9.0, and so deprecation warnings appear during tests. I would prefer to wait until all of our dependencies are fully compatible with 16.9.0, before upgrading.